apps/studio/lib/api/apiWrapper.test.ts

```
@@ -0,0 +1,147 @@
+    expect(apiAuthenticate).toHaveBeenCalledWith(mockReq, mockRes)
+    expect(mockHandler).not.toHaveBeenCalled()
+    expect(mockRes.status).toHaveBeenCalledWith(401)
+    expect(mockRes.json).toHaveBeenCalledWith({
+      error: {
+        message: 'Unauthorized: API error happened while trying to communicate with the server.',
+      },
+    })
+  })
+
+  it('should not authenticate when IS_PLATFORM is false', async () => {
+    ;(IS_PLATFORM as any) = false
```

Biome (2.1.2)

[error] The imported variable IS_PLATFORM is read-only

The variable is imported here

Use a local variable instead of reassigning an import.

(lint/suspicious/noImportAssign)